src/utils/versions/getVersionsColors.ts

```diff
@@ -0,0 +1,98 @@
+import type {VersionToColorMap, VersionsMap} from '../../types/versions';
+
+import {getMajorVersion, getMinorVersion} from './parseVersion';
+
+export const hashCode = (s: string) => {
+    return s.split('').reduce((a, b) => {
+        const num = (a << 5) - a + b.charCodeAt(0); // eslint-disable-line
+        return num & num; // eslint-disable-line
+    }, 0);
+};
+
+// 11 distinct colors from https://mokole.com/palette.html
+export const COLORS = [
+    '#008000', // green
+    '#4169e1', // royalblue
+    '#ffd700', // gold
+    '#ff8c00', // darkorange
+    '#808000', // olive
+    '#e9967a', // darksalmon
+    '#ff1493', // deeppink
+    '#00bfff', // deepskyblue
+    '#da70d6', // orchid
+    '#3cb371', // mediumseagreen
+    '#b22222', // firebrick
+];
+
+export const GRAY_COLOR = '#bfbfbf'; // --yc-color-base-neutral-hover
+
+export const getVersionsMap = (versions: string[], initialMap: VersionsMap = new Map()) => {
```

Comment:
I use old internal version of `getVersionsMap` and `getVersionToColorMap`, but with updated colours, to get version to color mapping